src/content/operations/monitoring/_index.en.md

```diff
@@ -20,38 +20,29 @@ The following metrics are exposed currently:
 
 * `submariner_gateways`: the number of gateways in the cluster
 
-* `submariner_gateway_creation_timestamp`: timestamp of gateway creation time, with the following labels:
+* `submariner_gateway_creation_timestamp`: timestamp of gateway creation time (labels: `local_cluster`, `local_hostname`)
 
-  * `local_cluster`: the local cluster name
-  * `local_hostname`: the local hostname
+* `submariner_gateway_sync_iterations`: gateway synchronization iterations
 
-* `submariner_connections`: the number of connections to other clusters, with the following labels:
```

Comment:
This is being renamed to `submariner_requested_connections`, and not merged with the new `submariner_connections`.

Reply:
Thanks @skitt. Can you help me understand where `submariner_requested_connections` is defined? I can't see it under
https://github.com/submariner-io/submariner/blob/devel/pkg/cable/metrics.go

Reply:
The change isn’t merged yet, see https://github.com/submariner-io/submariner-operator/pull/1081 — I think we should keep the two metrics separate.

Reply:
Thanks, makes sense to me.